pysb/simulator/base.py

```diff
@@ -356,6 +360,11 @@ def _process_incoming_initials(self, new_initials):
                                      'values'.format(cplx_pat))
         else:
             if not isinstance(new_initials, np.ndarray):
+                if not isinstance(new_initials, list):
```

Comment:
Could this nested-not bit be refactored? Maybe normalize list->ndarray up top where DataFrame is normalized to dict, then this whole `else` can become `elif isinstance(new_initials, np.ndarray)` and the ValueError can be moved to a final `else` at the bottom.